torax/run_simulation_main.py

```diff
@@ -35,6 +35,8 @@
 from torax.plotting import plotruns_lib
 from torax.torax_pydantic import model_config
 
+# Absorb pytest’s “--rootdir” flag so absl doesn’t fatally bail under pytest.
+flags.DEFINE_string('rootdir', None, 'Ignored pytest rootdir flag.')
```

Comment:
We should avoid defining logic in modules that are only needed for tests (unless we really need to!). I'm not entirely aware of the exact pathway that is causing this error, does the flag parsing that was previously done in the change https://github.com/google-deepmind/torax/commit/e41fd1b2c7d6804c4ae6831008454064759a993f, fix your problem?

If so that would be a nicer way to fix this as it fixes things in test logic.